### PR TITLE
Add all of the portfolio changes instructed by the the tutorial.

### PR DESCRIPTION
**Pull request by leah-mayo**
Personalize the portfolio by adding facts about myself, an image, and a short description to the portfolio. Also changed p text color to purple.

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -13,11 +13,11 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Adds a random fact to the page.
  */
-function addRandomGreeting() {
+function addRandomFact() {
   const greetings =
```
Comment (sperzion) [inline portfolio/src/main/webapp/script.js:19]: For readability, maybe change the greetings variable name to facts (and make the appropriate updates to the code below).

Comment (leah-mayo) [reply portfolio/src/main/webapp/script.js:19]: Ok, done. Should I add that change to this pull request (if that is possible) or just include it in the next one?

Comment (sperzion) [reply portfolio/src/main/webapp/script.js:19]: It should be possible to have a new commit or amend your current commit and include it as part of this pull request. Let me know if you can't find out how to do it, and I can help find a link with instructions.

Comment (leah-mayo) [reply portfolio/src/main/webapp/script.js:19]: I figured out how to use git commit --amend, but I had already pushed another set of changes so I'm not sure how to make the right changes appear here

Comment (sperzion) [reply portfolio/src/main/webapp/script.js:19]: Did you remember to push your branch to origin?

Comment (leah-mayo) [reply portfolio/src/main/webapp/script.js:19]: I think so, I ran git push origin LeahDevelopment

Comment (sperzion) [reply portfolio/src/main/webapp/script.js:19]: This seems to indicate that you don't need to do anything special other than pushing your branch.

https://github.community/t/do-i-need-to-make-a-new-pull-request-for-every-commit/13098

Did you maybe make the changes in a different branch, perhaps? Or haven't pushed after the latest update?

Comment (leah-mayo) [reply portfolio/src/main/webapp/script.js]: Wow, I forgot to push. The commit with the changes is there now. Thanks

**Pull request merged**